dataverse-webapp/src/test/resources/json/import/legacyHarvestedDataset.json

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
My reasoning was that since it is publicly available data there is no need to do it, since you don't even need an account to acquire such data. If someone can reach this file on github he can also gain access to the original. What do you think?